src/main/java/seedu/clinkedin/model/person/Person.java

```diff
@@ -105,6 +105,14 @@ public ObservableMap<TagType, UniqueTagList> getTags() {
         return tagTypeMap.asUnmodifiableObservableMap();
     }
 
+    /**
+     * Returns an immutable unique tag type map
+     * @return an immutable unique tag type map
+     */
+    public UniqueTagTypeMap getTagTypeMap() {
+        return tagTypeMap;
+    }
```

Comment:
Is it safe to this, since the `tagTypeMap` field of `Person` is not final?

Reply:
In this case, it's probably safe to return it because this function is tentatively being used for creating a copy of the person in testcases where it is not being modified (AddNoteCommand, AddLinkCommand etc)